addons/website_livechat/tests/test_chatbot_ui.py

```diff
@@ -97,3 +97,7 @@ def test_complete_chatbot_flow_ui(self):
 
     def test_chatbot_available_after_reload(self):
         self.start_tour("/", "website_livechat_chatbot_after_reload_tour", step_delay=100)
+
+    def test_chatbot_test_page_tour(self):
+        test_page_url = f"/chatbot/{'-'.join(self.chatbot_script.title.split(' '))}-{self.chatbot_script.id}/test"
+        self.start_tour(test_page_url, "website_livechat_chatbot_test_page_tour", login="admin")
```

Comment:
not as admin if possible (sudo won't be covered for example)

Reply:
done